litmus-portal/frontend/src/views/CreateWorkflow/ReliabilityScore/__tests__/ReliabilityScore.test.tsx

```diff
@@ -0,0 +1,23 @@
+import '@testing-library/jest-dom';
+import { act, screen } from '@testing-library/react';
+import React from 'react';
+import { render } from '../../../../testHelpers/test-util';
+import ReliabilityScore from '../index';
+
+describe('Reliability Score', () => {
+  it('has the right slider properties', async () => {
+    const ref = React.createRef();
+    act(() => {
+      render(<ReliabilityScore ref={ref} />);
+    });
+
+    await act(async () => {
+      await expect(screen.getByRole('slider').className).toBe(
```

Comment:
can you figure out if you can mimic a click event and actually increase and decrease the slider and then checking if the value and colours are coming accurately?
Recently there was a bug here that after a rerender the value was 10 but colour was yellow.

Reply:
As far as I know, when we mock `ReliabilityScore` we get access to the whole component with `Forward Refs`, which has Weight Slider which can be modified, but it only modifies when we are going with an end-to-end flow, thus to do any sort of mimicking we've to have an E2E approach with this entire page/view. 

To just mimic user events in the slider I have already written a test in `WeightSlider` which increases and decreases the slider value and checks for it.